models.py

```diff
@@ -26,17 +28,17 @@ class Request(Base):
     logo: Mapped[str]
     short_descr: Mapped[str]
     sites: Mapped[str]
-    contacts: Mapped[str]
-    socnets: Mapped[str]
+    contacts: Mapped[List[str]]
```

Comment:
```suggestion
    contacts: Mapped[list[str]]
```

We code on python 3.10+, so direct type hints are supported. I'll connect linter later to validate this automatically